File: Fibonacci and Factorial/Fibonacci_and_Factorial.py
def fib(n):
    if n==1:
        return 1
    elif n == 0:
        return 0
    else:
        return fib(n-1)+ fib(n-2)
    
def fac(n):
    if n==0 or n==1 :
        return 1
    else:
        return n*fac(n-1)



def main():
    n = int(input("Enter the value of n: "))
    #now call the functions to apply

if __name__ == "__main__":
    main()

File: fibonacci_and_factorial.c
#include<stdio.h>

int fact(int n){
   if(n ==1 || n ==0){
    return 1;
   }
   else {
    return  n * fact(n-1);
   }
   
}
// 0 1 1 2 3 5 8
int fibonacci(int n){
 if(n==1){
    return 1;
    }  
    else if(n == 0){
        return 0;
    } else {
    return fibonacci(n-1)+fibonacci(n-2);
 }
}

int main (){
    int n; 
    printf("Enter the value of n : \n");
    scanf("%d", &n);
   //call functions to apply now
}